### ISSUE TITLE
Swirl problem with the vector module

### ISSUE THREAD

**Issue by geanders** [geanders/RProgrammingForResearch#5]

**Comment by tburket** [comment]
> library(swirl)

| Hi! I see that you have some variables saved in your workspace. To keep things running smoothly, I recommend you
| clean up before starting swirl.

| Type ls() to see a list of the variables in your workspace. Then, type rm(list=ls()) to clear your workspace.

| Type swirl() when you are ready to begin.

> swirl()

| Welcome to swirl! Please sign in. If you've been here before, use the same name as you did then. If you are new,
| call yourself something unique.

What shall I call you? Tori

| Would you like to continue with one of these lessons?

1: R Programming Vectors
2: No. Let me start something new.

Selection: 1

| What do you think tf will look like?

1: a single logical value
2: a vector of 4 logical values

Selection: 1
Error in swirlExpectation(condition(object)) : 
  could not find function "is_equivalent_to"

| Leaving swirl now. Type swirl() to resume.

> swirl()

| Welcome to swirl! Please sign in. If you've been here before, use the same name as you did then. If you are new,
| call yourself something unique.

What shall I call you? Tori

| Would you like to continue with one of these lessons?

1: R Programming Vectors
2: No. Let me start something new.

Selection: 1

| What do you think tf will look like?

1: a vector of 4 logical values
2: a single logical value

Selection: 1
Error in swirlExpectation(condition(object)) : 
  could not find function "is_equivalent_to"

| Leaving swirl now. Type swirl() to resume.

**Comment by leighseverson** [comment]
the "Vectors" lesson in the R_Programming course doesn't have this same error - might have to do with swirl's `omnitest` function?

**Comment by geanders** [comment]
Hmmm. I checked the code, and R Programming is also using `omnitest` to test the answer for that question. Let me see if those functions are different in the two courses, though.

**Comment by geanders** [comment]
Okay, I'm on the path-- It's definitely something to do with `omnitest`, and from the commit history from R_Programming, it looks like it's been fixed in that main swirl course, but I'm guessing the developers didn't make the same patch to the R_Programming_Alt. I've emailed one of the swirl folks to make sure I understand what's going on with that function before I try to put in a fix. It looks like they ultimately deleted the `omnitest` definition from custom tests in R_Programming, but I wanted to figure out where the right version of the function is defined, to make sure that R_Programming_Alt will be able to find that if I delete the "customTests" function definition. @seankross

**Comment by seankross** [comment]
Use this course from now on: https://github.com/swirldev/R_Programming_E

**Comment by geanders** [comment]
You're the greatest, @seankross -- thanks!!

@leighseverson would you change the homework assignment in the course ebook to change which course they should use? ("R_Programming_E" instead of "R_Programming_Alt"), and we'll let the class know. We'll count full credit for any R_Programming_Alt lessons that people have already done, but use this "E" version going forward.

**Comment by seankross** [comment]
There are new installation instructions as well:

``` r
library(swirl)
install_course_github("swirldev", "R_Programming_E")
swirl()
```

**Comment by geanders** [comment]
Noted. Thanks!

**Comment by leighseverson** [comment]
Thank you @seankross ! 

@geanders - great, everything should be updated now!
